docs/source/development/fingerprints_in_homeassistant.rst

```diff
@@ -1,52 +1,31 @@
 Using fingerprints in Home Assistant
 ====================================
-Sometimes it can be useful to load the **fingerprints** used for our **pytest suite** in the development of the Home Assistant component. 
+Sometimes it can be useful to load the **fingerprints** used for our **pytest suite** in the development of the Home Assistant component.
+This enables debugging of the UI in Home Assistant which is not possible from pytest alone.
 
-Set up bimmer_connected
------------------------
-Clone the repository to ``~/bimmer_connected``:
+.. warning::
+  If not already done, `set up the Home Assistant development environment <https://developers.home-assistant.io/docs/development_environment>`_.
 
-.. code-block:: bash
+Now start Home Assistant at least once and let all python packages install (``hass -c ./config``).
+If not already done, set up the **BMW Connected Drive Integration** in Home Assistant.
+Shut down Homeassistant afterwards.
 
-   git clone https://github.com/bimmerconnected/bimmer_connected.git ~/bimmer_connected
+Now, we have to "hack" our mocked backend calls into Home Assistant.
 
 .. note::
-  
-  Make sure that you can access this path from inside your Home Assistant virtual environment!
+  Doing so will remove your own account!
 
-Edit the  ``_get_vehicles()`` function in `account.py#L370-L392 <https://github.com/bimmerconnected/bimmer_connected/blob/master/bimmer_connected/account.py#L370-L392>`_
-and add the following code between ``self._get_oauth_token()`` and ``for brand in CarBrand:``:
+Edit ``homeassistant/components/bmw_connected_drive/coordinator.py`` and locate the function ``_async_update_data``. We now have to replace ``await self.account.get_vehicles()``. The ``try .. except`` block should look like this:
 
```

Comment:
This folder is not available on HA OS or Supervised installs, so good to mention that this is available on Container / Core installs (don't know what your install type is)

Reply:
Basically the whole thing is referencing the HA dev environment (i.e. running it on your local machine in VS Code). Can I make this somehow clearer with the existing warning to the HA dev environment?

Would you run this also in your local HAOS installation (it would surely mess up a lot of settings)?

Reply:
Ah I see. then good to mention in the title that this is for Home Assistant develeopment environment. That should be clear then.

Doesn't sound logic to run this in a local HAOS installation.

Reply:
The warning should be clear enough?

![image](https://github.com/bimmerconnected/bimmer_connected/assets/42204099/6f995eaf-23a1-4a78-aae5-57e1221888df)